src/CattyTests/Views/BrickCells/Data/BrickCellVariableDataTests.swift

```diff
@@ -36,7 +36,7 @@ class BrickCellVariableDataTests: XCTestCase {
     var variablesContainer: VariablesContainer!
     var objectVariable1: UserVariable!
     var objectVariable2: UserVariable!
-    var secondObjectVariable: UserVariable!
+    var secondObjectVariable: UserList!
```

Comment:
Please change `secondObjectVariable` to `UserVariable` (also in the `setUp`).

Reply:
Hi @m-herold  `secondObjectVariable` is added as `objectList` using `addObjectList` should I change that to `addObjectVariable`

https://github.com/Catrobat/Catty/blob/13ed308fc66e7b464aefbf42635057a2700c0ca0/src/CattyTests/Views/BrickCells/Data/BrickCellVariableDataTests.swift#L58

Reply:
Yes, please.